docs/content/contribute/index.md

```diff
@@ -1,48 +1,109 @@
-# Contributing to the Docs
+# 🚀 Contributing to the Docs
 
-We welcome contributions to our documentation! If you would like to contribute, please follow the steps below.
+Welcome to our documentation contribution guide! We're excited to have you here. This guide will help you get started with contributing to our documentation. Let's make our docs better together! 💪
 
-## Setting up the Docs
+<div align="center">
 
-1. Clone the repository:
+![Documentation Contributors](https://img.shields.io/github/contributors/Significant-Gravitas/AutoGPT?style=for-the-badge)
+![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=for-the-badge)
+![Documentation Build Status](https://img.shields.io/badge/docs-passing-success?style=for-the-badge)
```

Comment:
I like the idea process behind this but lets remove them for now since they don't work in the preview build (https://deploy-preview-9037--auto-gpt-docs-dev.netlify.app/contribute/)

![Uploading image.png…]()